docs/source/get_started/model_data/model_data.rst

```diff
@@ -2595,6 +2595,79 @@ copy migration files from installed modules to the :code:`.motech` directory. Fi
 directory(if you are using mysql then use :code:`mysql` instead :code:`default`) in the bundle. Each file muse have
 `a proper name <http://flywaydb.org/documentation/migration/sql.html>`_(e.g. :code:`V1__Description.sql`).
 
+####################
+Schema Import/Export
+####################
+
+This section explains how to `import`_ and `export`_ MDS Schema. It can be done with only few steps using the MDS module
+UI.
+
+Schema Import
+#############
+
+The first thing you need to do is to enter Import tab under the Settings.
+
+            .. image:: img/import_schema_tab.png
+                    :scale: 100 %
+                    :alt: MDS Import Schema tab
+                    :align: center
+
+Once in the Import tab, you will have to click the "Select file" button and choose the file you want to import the
+schema from. It must be a valid JSON file that contains a valid schema definition, otherwise an error will occur while
+parsing it.
+
+            .. image:: img/submit_form.png
+                    :scale: 100 %
+                    :alt: MDS Import submit form
+                    :align: center
+
+Here, all you have to do is to click the Submit button. MDS will now parse the JSON file and display entities for review
+on the following screen.
+
+            .. image:: img/import_selection.png
+                    :scale: 100 %
+                    :alt: MDS Import selection
+                    :align: center
+
+Now you can select which entities you want to import. You can achieve this by simply ticking or unticking the checkboxes
+next to the entity or module name in the Schema column. Ticking the checkbox next to the module name will result in
+importing all the entities originating from that module, while ticking the checkbox next to the entity name will result
+in importing only that entity. If the given JSON contains instances of the entities you are importing schema for, you
+can also import them by ticking the checkboxes next to the entity or module name in the data column. Once you have
+selected all the entities you want to import schema and/or instances for, simply click the Import button. Keep in mind
+that importing schema for entities that already exist will result in removing data related to those entities. Two
+entities are equal if they have the same class name. For information on how relationships are handled please visit
```

Comment:
change to "are considered equal" (in reality they might not be equal, so we want to emphasize what we consider equal in this context)